src/main/java/step3/LottoGenerator.java

```diff
@@ -1,16 +1,19 @@
 package step3;
 
+import step3.VO.LottoNumber;
+
 import java.util.ArrayList;
 import java.util.Collections;
 import java.util.List;
 
 public class LottoGenerator {
+    public final static int lottoSize = 6;
```

Comment:
상수 치환 👍  제한자 순서는 다음 링크를 참고해주시면 감사하겠습니다 :)
```suggestion
    public static final int lottoSize = 6;
```

https://naver.github.io/hackday-conventions-java/#modifier-order